cmake/serac-config.cmake.in

```diff
@@ -139,7 +141,8 @@ if(NOT SERAC_FOUND)
 
   # Petsc
   if(SERAC_USE_PETSC)
-    # Note: Targets not currently imported
+    # Use the installed FindPETSc.cmake
+    find_dependency(PETSc)
```

Comment:
Can you just call include(FindPetsc.cmake) here?

Reply:
pretty sure. ill change that.

Reply:
changed to:

```cmake
include("${CMAKE_MODULE_PATH}/FindPETSc.cmake")
```